content/adopting-an-ethical-license.md

```diff
@@ -0,0 +1,41 @@
++++
+title = "Adopting an Ethical License"
+heading = "Adopting an Ethical License"
++++
+
+If you are starting a new project, you _may_ [review existing ethical licenses](./licenses/) and choose the one that most directly matches your goals.
+
+For maintainers considering adopting an ethical license on an existing project, be aware that this requires significant work. Each license is different, which makes a general guide difficult to write. Bear in mind that this is not legal advice, and we are not lawyers.
+
+Because there is significant social risk associated with relicensing, we recommend starting by centering an intent to preserve and further develop the trust between maintainers, contributors, and users of the project.
+
+From there, an understanding of the authorship rights and license constraints will serve maintainers well. To do so, we recommend reaching a high degree of confidence in the precision and accuracy of your answers to the following questions:
```

Comment:
I'd restate this whole paragraph as:

Before changing your project's licensing, you need to understand exactly how it works today. Consider:

Reply:
I would prefer a strong assertion that people get clear answers to these two questions, as having good answers is the difference between a well-executed relicensing (or sub/dual-licensing) effort and one that draws valid complaints of erasure and appropriation.

Reply:
IMO the social risks were clearly stated in the previous paragraph. 🤷‍♂

Reply:
True, I'll tone it down in next draft :)